src/sentry/workflow_engine/handlers/condition/age_comparison_handler.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any
+
+from django.utils import timezone
+
+from sentry.rules.age import AgeComparisonType, age_comparison_map
+from sentry.rules.filters.age_comparison import timeranges
+from sentry.workflow_engine.models.data_condition import Condition
+from sentry.workflow_engine.registry import condition_handler_registry
+from sentry.workflow_engine.types import DataConditionHandler, WorkflowJob
+
+
+@condition_handler_registry.register(Condition.AGE_COMPARISON)
+class AgeComparisonConditionHandler(DataConditionHandler[WorkflowJob]):
+    @staticmethod
+    def evaluate_value(job: WorkflowJob, comparison: Any) -> bool:
+        event = job["event"]
+        first_seen = event.group.first_seen
+        current_time = timezone.now()
+        comparison_type = comparison.get("comparison_type")
```

Comment:
@cathteng @ameliahsu 🤔 thoughts on how we could add type safety to these? i think this is fine for now, but i wonder if we could improve this system a bit.

One idea I had was to add another generic type on the condition handler, then pass it through to evaluate value (the same way we are for the first parameter in evaluate value).

Any ideas on how we could do that with json schema or anything? 🤔

Reply:
i was thinking of validation on the `comparison` using a dataclass with types given the `DataCondition.type` with a registry storing the dataclasses. this would be more robust than json schema because you can use a list of possible values for a single key, whereas json schema is limited to the basic types

Reply:
basically spreading the comparison dict into the dataclass to check that the contents we need are there and are the correct type